schedule/hanaperf/hana_install.yaml

```diff
@@ -1,135 +1,34 @@
 ---
-name: hanaperf_installation
-description: >
-  Maintainer: dawei.pang@suse.com
-  OS installation for HANA performance
+name: hana_ipxe_autoyast
```

Comment:
there was a file named `hana_ipxe_autoyast.yaml` that has been deleted and this file has been sort of converted to do what the other file did. Am I correctly understand this?

Reply:
Sorry, this is my mistake. I will correct the naming.

Reply:
fixed